deploy/cr.yaml

```diff
@@ -41,6 +41,11 @@ spec:
   #
   # useImmutableImage: true
 
+  # [On development, to be released]
+  # If enabled this will automatically use the webhook for injecting into applications
```

Comment:
No, it won't. It will only stop the OneAgent Pod from doing it. The customer still needs to use OneAgentAPM.

Also, do we want this here now? It's still a work in progress on the agent/cluster side.